apps/auth/db/commands.py

```diff
@@ -27,13 +27,12 @@ class CreateUserCommand(superdesk.Command):
         superdesk.Option('--username', '-u', dest='username', required=True),
         superdesk.Option('--password', '-p', dest='password', required=True),
         superdesk.Option('--email', '-e', dest='email', required=True),
-        superdesk.Option('--admin', '-a', dest='admin', required=False),
+        superdesk.Option('--admin', '-a', dest='admin', required=False, default=False),
```

Comment:
that probably works if you don't use `Option` and it does config automatically
btw `dest` should be `is_admin`